L1Trigger/Configuration/python/SimL1Emulator_cff.py

```diff
@@ -151,9 +151,9 @@
 from L1Trigger.L1TTrackMatch.l1tTrackerEtMiss_cfi import *
 from L1Trigger.L1TTrackMatch.l1tTrackerHTMiss_cfi import *
 # make the input tags consistent with the choice L1VertexFinder above
-l1tTrackJets.L1PVertexCollection  = ("l1tVertexFinder", "l1vertices")
+l1tTrackJets.VertexInputTag  = ("l1tVertexFinderEmulator","l1verticesEmulation")
 l1tTrackFastJets.L1PrimaryVertexTag  = ("l1tVertexFinder", "l1vertices")
-l1tTrackJetsExtended.L1PVertexCollection  = ("l1tVertexFinder", "l1vertices")
+l1tTrackJetsExtended.VertexInputTag  = ("l1tVertexFinderEmulator","l1verticesEmulation")
 #l1tTrackerEtMiss.L1VertexInputTag = ("l1tVertexFinder", "l1vertices")
 #l1tTrackerEtMissExtended.L1VertexInputTag = ("l1tVertexFinder", "l1vertices")
```

Comment:
Are these commented lines necessary? Can they be removed?

Reply:
I did not put those... they were there. I can remove them though